bioinf_modules/bio_files_processor.py

```diff
@@ -0,0 +1,48 @@
+import os
+""" This is python module that has two functions
+convert_multiline_fasta_to_oneline and parse_blast_output
+"""
+
+
+def convert_multiline_fasta_to_oneline(input_fasta: str,
+                                       output_fasta='output_fasta'):
+    """ Function to convert input multiline
+    fasta-file to oneline output fasta-file.
+    Input is filename in the current directory,
+    Output will be created in the same directory with default name output_fasta
+    """
+    full_path = os.path.abspath(__file__)
+    path = os.path.dirname(full_path)
+
+    with open(path + '/' + input_fasta, 'r') as in_fasta:
+        with open(path + '/' + output_fasta, 'a') as out_fasta:
```

Comment:
Тут можно тоже с os.path.join
```suggestion
    with open(os.path.join(path, input_fasta), 'r') as in_fasta:
        with open(os.path.join(path, output_fasta) as out_fasta:
```